interactions/api/models/guild.py

```diff
@@ -1600,6 +1600,21 @@ async def search_members(self, query: str, limit: Optional[int] = 1) -> List[Mem
         )
         return [Member(**member, _client=self._client) for member in res]
 
+    @property
+    def icon_url(self) -> str:
+        """
+        Returns the URL of the guild's icon.
+
+        :return: URL of the guild's icon (None will be returned if none of icon is set)
+        :rtype: str
+        """
+        url = "https://cdn.discordapp.com/"
```

Comment:
`url`'s declaration here is unnecessary- this line could be removed, and L613 could be changed to include this "base" part instead (as well as assigning the variable instead of adding to it).

Reply:
@Astrea49 I think this is fine, since all the ``avatar_url`` or ``icon_url`` property in the library itself is doing the same thing. Check out the ``avatar_url`` under ``api.models.user.User``.

Reply:
It's a *bit* different there - an icon with a value of `None` there means there's still *an* icon for that user (the default Discord ones), while a `None` value for guilds means, well, it has no icon, being represented by its initials instead (which in this case don't count as icons).

There, no matter what, the "base" URL is needed for every part of the if statement, while it's only needed in one part here.

(Ultimately, this *is* nitpicking a bit, but it would play nicer with some typehinters. It's not a *strictly* necessary change.)

Reply:
Ah, I see. Will be fixed it in a moment after I test stuff again.